### PR TITLE
adding test for methods get and count

### PR DESCRIPTION
**Pull request by dairof7**
adding test for methods get and count in file storage and db storage

### CODE REVIEW

File: tests/test_models/test_engine/test_db_storage.py [hunk 1]
```diff
@@ -78,11 +79,33 @@ def test_all_returns_dict(self):
...
+        self.assertEqual(type(get_obj), State)
+
+    @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
+    def test_count(self):
+        """ Test case for count method at state"""
+        first_count = storage.count(State)
+        dic_new_state = {'name': 'Valle'}
+        new_obj = State(**dic_new_state)
+        new_obj.save()
+        second_count = storage.count(State)
```
Comment (bryano13) [inline tests/test_models/test_engine/test_db_storage.py:110]: How is this different from first count?

Comment (dairof7) [reply tests/test_models/test_engine/test_db_storage.py:110]: the first count is before create a new state, and the second count is after create it, so we can compare it to verify if the counter increase

Comment (bryano13) [reply tests/test_models/test_engine/test_db_storage.py:110]: Ok, its clear now, thx!

**Comment by bryano13** [comment]
Tests

**Pull request merged**